### ISSUE TITLE
SRARS-COV-2 Nsp15 RA-0010874 2nd Hit expansion

### ISSUE THREAD

**Issue by qxsml** [StructuralGenomicsConsortium/CNP12-Nsp15-TAMU#7]
After testing the 21 compounds and analyzing the results, please find the summary below. 
![image](https://github.com/StructuralGenomicsConsortium/CNP12-Nsp15-TAMU/assets/147162201/872fbc74-6e0e-470a-89da-50ff5b6851fc)
![image](https://github.com/StructuralGenomicsConsortium/CNP12-Nsp15-TAMU/assets/147162201/9ad34c65-6ec1-4c49-b5c8-fe107898f00a)

A new series of compounds has been planned based on these results and is currently being synthesized for a second expansion.
Please see below:
![image](https://github.com/StructuralGenomicsConsortium/CNP12-Nsp15-TAMU/assets/147162201/03bab63d-8a57-45bf-bebf-171373f799fc)

**Comment by rahmanszsaleem** [comment]
Excellent work. 
Looking at the commonalities between the two compounds 
![image](https://github.com/StructuralGenomicsConsortium/CNP12-Nsp15-TAMU/assets/147126062/05e5d837-e1ff-4035-a224-ef7d1647a90c)

I am tempted to suggest that we should also prepare/purchase and test the following as derivatives of RA-0010874
 
![image](https://github.com/StructuralGenomicsConsortium/CNP12-Nsp15-TAMU/assets/147126062/53709d20-f0ae-4488-b044-f6178e81182e)

X= H, COOH, OH
Further, we may also substitute the heterocycle attached to quinoline with other N-heterocycles (depending on the commercial availability).

**Comment by qxsml** [comment]
> Excellent work. Looking at the commonalities between the two compounds ![image](https://private-user-images.githubusercontent.com/147126062/329627294-05e5d837-e1ff-4035-a224-ef7d1647a90c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzNTQxNDYsIm5iZiI6MTcxNTM1Mzg0NiwicGF0aCI6Ii8xNDcxMjYwNjIvMzI5NjI3Mjk0LTA1ZTVkODM3LWUxZmYtNDAzNS1hMjI0LWVmN2QxNjQ3YTkwYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMFQxNTEwNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMzg1YzYzZTVkODgyMzc0OGIyNGM2YTNhZWZmMDhmZTQ3NDlhNzBmM2NiZGI2NTI2ZTZiOTZiMzVlNjUxZTU5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.yK3SJMaOKnwaG98nV2-6RtAE8ctPjGe3llZM-iKkmsw)
> 
> I am tempted to suggest that we should also prepare/purchase and test the following as derivatives of RA-0010874
> 
> ![image](https://private-user-images.githubusercontent.com/147126062/329627370-53709d20-f0ae-4488-b044-f6178e81182e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzNTQxNDYsIm5iZiI6MTcxNTM1Mzg0NiwicGF0aCI6Ii8xNDcxMjYwNjIvMzI5NjI3MzcwLTUzNzA5ZDIwLWYwYWUtNDQ4OC1iMDQ0LWY2MTc4ZTgxMTgyZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMFQxNTEwNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNjUxNWM5OTEyODg1MDg5ZGI2NmQwODk1NzdhODlmMjAwNWI4M2FhMWUzN2ViNWI4NzMzMzQyY2ExNTc5ZTBhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Z2-2F2kbQCXt1kDd76pI1mK_la0nsu4OWrsCBeQNstk)
> 
> X= H, COOH, OH Further, we may also substitute the heterocycle attached to quinoline with other N-heterocycles (depending on the commercial availability).

Hi Rahman, thank you very much for your suggestion, X=H form is totally doable, SMs are cheap. I'm keen to try conditions in this paper https://doi.org/10.1039/C9OB00033J, very interesting. I'll leave the OH version for the 3rd expansion if the H version gave promising results.

**Comment by rahmanszsaleem** [comment]
Yes, a wonderful synthetic protocol for the suggested molecules!
